tests/requirements.txt

```diff
@@ -1,3 +1,4 @@
 # test libraries
 coverage
 djangocms_helper
+factory-boy
```

Comment:
I wasn't sure what I needed to pull across from Versioning regarding the test factories. The test_requirements list could be wrong too. My mistake.